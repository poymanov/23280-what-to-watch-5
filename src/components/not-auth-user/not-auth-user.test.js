import React from "react";
import renderer from "react-test-renderer";
import NotAuthUser from "./not-auth-user";
import {Router as BrowserRouter} from "react-router-dom";
import browserHistory from "../../etc/browser-history";

it(`Should NotAuthUser render correctly`, () => {
  const tree = renderer
    .create(<BrowserRouter history={browserHistory}><NotAuthUser /></BrowserRouter>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
