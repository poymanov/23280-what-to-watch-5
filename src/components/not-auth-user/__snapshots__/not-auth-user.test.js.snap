// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should NotAuthUser render correctly 1`] = `
<div
  className="user-block"
>
  <a
    className="user-block__link"
    href="/login"
    onClick={[Function]}
  >
    Sign in
  </a>
</div>
`;
